src/lang-promql/complete/hybrid.ts

```diff
@@ -59,25 +59,26 @@ const autocompleteNode: { [key: string]: AutoCompleteNode } = {
 
 const snippets: readonly SnippetSpec[] = [
   {
-    keyword: 'sum(rate(__input_vector__[5m]))',
+    label: 'sum(rate(__input_vector__[5m]))',
+    type: 'function',
     snippet: 'sum(rate(${__input_vector__}[5m]))',
   },
   {
-    keyword: 'histogram_quantile(__quantile__, sum by(le) (rate(__histogram_metric__[5m])))',
+    label: 'histogram_quantile(__quantile__, sum by(le) (rate(__histogram_metric__[5m])))',
+    type: 'function',
     snippet: 'histogram_quantile(${__quantile__}, sum by(le) (rate(${__histogram_metric__}[5m])))',
   },
   {
-    keyword: 'label_replace(__input_vector__, "__dst__", "__replacement__", "__src__", "__regex__")',
+    label: 'label_replace(__input_vector__, "__dst__", "__replacement__", "__src__", "__regex__")',
+    type: 'function',
```

Comment:
It seems this `type` is not propagated into the `parsedSnippets` and thus not used anywhere? Not sure it should be `function` though. Should we invent our own `snippet` type?

Reply:
And then if it's always `snippet` for snippets, I guess we can just hardcode it in the `parsedSnippets` below.

Reply:
It would be nice indeed to a have a dedicated icon for the snippet !

I will review the snippet management it seems I was a bit too fast here

Reply:
> And then if it's always snippet for snippets,

What do you mean by that ? Not sure I got your point actually :(.

Reply:
I simplify the way to manage the snippet, hope you thought it like that too !

Reply:
Ah ok yeah, what you have now is fine too.